climada_petals/test/test_wildfire.py

```diff
@@ -110,7 +162,7 @@ def test_proba_fire_season_pass(self):
     def test_summarize_fires_to_seasons_pass(self):
         """ Test probabilistic set_probabilistic_event_year_set """
         wf = WildFire()
-        wf.set_hist_fire_FIRMS(TEST_FIRMS)
+        wf = wf.from_hist_fire_FIRMS(TEST_FIRMS)
```

Comment:
Delete prior line and initialize like so:
```suggestion
        wf = WildFire.from_hist_fire_FIRMS(TEST_FIRMS)
```